plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraAnimationsPluginImpl.kt

```diff
@@ -778,6 +790,17 @@ internal class CameraAnimationsPluginImpl : CameraAnimationsPlugin {
     animatorSet.start()
   }
 
+  override fun onSizeChanged(width: Int, height: Int) {
+    mapCenter = ScreenCoordinate((width / 2).toDouble(), (height / 2).toDouble())
+    cameraAnchor = mapCenter
+  }
```

Comment:
Would be great to document this relationship.

Reply:
I remembered our discussion to send `null` as anchor to core - if I understand correctly it's equivalent to setting to map center as it is done now.
Will check it out, if it will work - it will simplify PR a bit, we will not need to override `onSizeChanged` at all.

Reply:
Not relevant anymore.